server/likes/views.py

```diff
@@ -62,11 +97,46 @@ def get(self, request, *args, **kwargs):
         for item in items:
             data.append(LikeSerializer(item).data)
 
-        return Response(data)
+        return Response({'type': 'likes', 'items': data})
+    
+    # for getting likes for remote_comment (requires 'comment_url' in the body)
+    def post(self, request, *args, **kwargs):
+        comment_url = request.data.get('comment_url')
+        if comment_url == None:
+            return Response({'error': 'Getting remote likes requires comment_url in the body'}, status=status.HTTP_400_BAD_REQUEST)
+
+        parsed_uri = urlparse(comment_url)
+        object_host = '{uri.scheme}://{uri.netloc}/'.format(uri=parsed_uri)
+
+        post_owner = self.kwargs['author_id']
+        post_id = self.kwargs['post_id']
+        comment_id = self.kwargs['comment_id']
+
+        if 'team6' in object_host:
```

Comment:
I think we can clean this logic up by creating a static var instead of using it as a magic string. Do you mind adding a `TODO: Clean this up `